data-raw/tidyverse_creation.R

```diff
@@ -0,0 +1,177 @@
+library(dplyr)
+# ------------------------------------------------------------------------------
+body_function <- function(fct, argument, args1) {
```

Comment:
The `argument` argument is never used in the body of this function. Please delete.